### PR TITLE
Fixed the display of documentation site. Increased to full width.

### LINKED ISSUE #179: Width of the documentation site covers only half the screen
/kind documentation  

**What happened**:
The documentation site at https://kubeedge.readthedocs.io  displays content in only half the screen.
 
**What you expected to happen**:
The width of the site should be full screen and  content should be displayed appropriately.  

**How to reproduce it (as minimally and precisely as possible)**:
Open the site https://kubeedge.readthedocs.io.

### PR DESCRIPTION
**Pull request by anvithks**
**What type of PR is this?**  

 /kind documentation

**What this PR does / why we need it**:
This is to fix the issue of the site display width.  

**Which issue(s) this PR fixes**:
Fixes #179 

**Special notes for your reviewer**:
[Before](https://kubeedge.readthedocs.io/en/latest/guides/message_topics.html) and
[After](https://kubeedge-anvith.readthedocs.io/en/latest/guides/message_topics/)

### CODE REVIEW

**Comment by anvithks** [comment]
@rohitsardesai83  Please review and merge.

**Comment by anvithks** [comment]
@pavan187  the label has not been added. I used the /kind documentation

**Comment by pavan187** [comment]
> @pavan187 the label has not been added. I used the /kind documentation

@anvithks Currently Ci-Bot, can add the /kind documentation from comment section, PR describe section handling is under progress

File: docs/_static/css/custom.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.wy-nav-content{
+    max-width: 100%;
+}
+.rst-content{
+    width: 75%;
+}
```
Comment (m1093782566) [inline docs/_static/css/custom.css]: Please add a newline.

Comment (anvithks) [reply docs/_static/css/custom.css]: @m1093782566 Do you want me to add a newline at the end of the file?  
Or is it in between the two styles?

Comment (m1093782566) [reply docs/_static/css/custom.css]: At the end of file please :)

Comment (anvithks) [reply docs/_static/css/custom.css]: Done.

**Comment by anvithks** [comment]
@rohitsardesai83 @m1093782566 Please review and merge.

**Comment by anvithks** [comment]
@m1093782566 Please review and approve.

**Pull request merged**